Header header
octomap_msgs/Octomap[] octomaps
string[] owners
float64[] sizes
uint8 num_octomaps
